tests/README.md

```diff
@@ -105,18 +111,34 @@ tests/scripts/helm.sh up
 
 Some settings are available to run the tests under different environments. The settings are all configured with environment variables.
 See [environment.go](/tests/framework/installer/environment.go) for the available environment variables.
+At least you should set the following variables.
+
+```console
+export TEST_HELM_PATH=/tmp/rook-tests-scripts-helm/linux-amd64/helm
+export TEST_BASE_DIR=WORKING_DIR
+export TEST_SCRATCH_DEVICE=<block device> # for example, TEST_SCRATCH_DEVICE=/dev/sdb
+```
+
+Please note that the integration test erases the contents of TEST_SCRATCH_DEVICE.
 
 To run all integration tests:
+```console
+_output/tests/linux_amd64/integration -test.v -test.timeout 1800s
```

Comment:
Why not keep `go test` instead of `_output/tests/linux_amd64/integration`? Then if you have any local changes to the test, it will be re-compiled automatically. Personally, I'll still use `go test`.

Reply:
Oops, sorry. It's just because I've used _output/tests/linux_amd64/integration in my environment. I'll send a PR to use `go test` after confirming everything works well.

Reply:
@travisn I'm considering how to set the path of test scripts like "tests/scripts/localPathPV.sh". In CI, pwd is the top directory of Rook repository. However, it's "tests/integration/" in `go test ...`. Do you have any good idea of how to set the proper path in both cases?

Reply:
@satoru-takeuchi Where exactly do you need to set it? If you're documenting it here, let's assume contributors are running it locally with the tests/integration path. Anywhere else we can then set it as needed for the CI, right?

Reply:
@travisn Thank you for your advice, I found `findRookRoot` is what exactly I wanted.

https://github.com/rook/rook/blob/master/tests/framework/installer/ceph_installer.go#L154

What I meant in the last comment is that the following code doesn't work in both CI environment and local environment.

```
const (
  cmd = "tests/scripts/localPathPV.sh"
)

func foo() {
  ...
  os.Exec(cmd)
  ...
}
```

It works well in the CI environment. In this case, the integration test is run via "_output/tests/linux_amd64/integration", pwd is Rook's top directory, and "tests/scripts/localPathPV.sh" exists. However, it doesn't when running the integration test by "go test ..." in the local environment. Here pwd would be "~/go/src/github.com/rook/rook/tests/integration" and then `os.Exec(cmd)' fails because "~/go/src/github.com/rook/rook/tests/integration/tests/scripts/localPathPV.sh" doesn't exist. However, findRookRoot resolves it.

Reply:
Yes, sounds good to know the `findRookRoot()` helper resolves it.